modules/gapi/test/gapi_util_tests.cpp

```diff
@@ -4,13 +4,36 @@
 //
 // Copyright (C) 2018 Intel Corporation
 
-
+#include <type_traits>
 #include "test_precomp.hpp"
+#include <opencv2/gapi/gtype_traits.hpp>
+#include <opencv2/gapi/util/util.hpp>
 
-#include <type_traits>
+namespace cv
+{
+struct Own {};
+namespace gapi
+{
+namespace own
+{
+struct ConvertibleToOwn{};
+struct NotConvertibleToOwn{};
 
-#include <opencv2/gapi/util/util.hpp>
+static cv::Own to_ocv(const cv::gapi::own::ConvertibleToOwn&) {
```

Comment:
Do we need this `to_ocv` here? Looks like all convertible etc stuff can be removed from tests and messages since we jumped to another approach